UPGRADE-4.3.md

```diff
@@ -183,19 +178,22 @@ Workflow
    ```yaml
    framework:
        workflows:
+           type: workflow
            article:
                marking_store:
                    type: multiple
+                   arguments: states
```

Comment:
I would stick to the default value instead

Reply:
Actually this is nice. It's better to fully understand if the property will contain a single place, or many places :+1: